### DISCUSSION TITLE
Windows API used for desktop icons and others

### DISCUSSION THREAD

**Discussion by nsantiago2719** [rocksdanister/lively#2042] [General]
Hi,

I have found this project while looking for a reference on my learning mini-project that I'm doing to learn about creating windows application. I am fairly new on creating desktop applications also with windows apis so I just want to ask, on which part of the code handles the desktop icons so that lively is always behind those icons.  Another thing is making sure that lively won't minimize when `Show Desktop` is executed(eg. via Windows + D). 

I was able see the `SetWindowsPos` from an windows api but that isn't enough(maybe) because it only set the position behind all applications but not the desktop icons.

I hope that someone can point me to the part of the code or some documentation I could read to achieve those. Thanks!

**Comment by rocksdanister** [comment]
https://www.codeproject.com/Articles/856020/Draw-Behind-Desktop-Icons-in-Windows-plus
